src/auth/mock-auth.guard.ts

```diff
@@ -16,7 +16,13 @@ export class MockAuthGuard {
   async canActivate(context: ExecutionContext) {
     const req = context.switchToHttp().getRequest();
     if (!this.user) {
-      this.user = await this.usersService.createUser('hardcoded', 'Testy');
+      // this assures that we can create the user 'hardcoded', if we need them before any calls are made or
```

Comment:
Please move this fix into a separate commit

Reply:
done